docs/en/ingest-management/integrations/install-integration-assets.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[install-uninstall-integration-assets]]
+= install and uninstall {agent} integration assets
+
+{agent} integrations come with a number of assets, such as dashboards, saved
+searches, and visualizations for analyzing data. When you add an integration
+to an agent policy, the assets are installed automatically. If you're building
+a configuration file by hand, you might need to install the assets.
```

Comment:
If you're building a configuration file by hand, you need to install required assets such as index templates.